src/content/en/updates/posts/2016/04/intersectionobserver.md

```diff
@@ -0,0 +1,164 @@
+As mentioned earlier, the callback will be triggered a single time when the
+observed element comes partially into view and another time when it has left
+the viewport. This way `IntersectionObserver` gives you an answer to the question, “Is element X in view?”. In some use
+cases, however, that might not be enough.
+
+That’s where the [`threshold`](https://wicg.github.io/IntersectionObserver/#dom-intersectionobserverinit-threshold) option comes into play. It allows you to define an array of [`intersectionRatio`](https://wicg.github.io/IntersectionObserver/#dom-intersectionobserverentry-intersectionratio) thresholds. Your callback will be called every time `intersectionRatio` crosses one of these values. The default value for `threshold` is `[0]`, which explains the default behavior. If we change `threshold` to `[0, 0.25, 0.5, 0.75, 1]`, we will get notified every time an additional quarter of the element becomes
+visible:
+
+<img src="/web/updates/images/2016/04/intersectionobserver/threshold.gif">
+
+# Any other options?
+As of now, there’s only one additional option to the ones listed above. [`rootMargin`](https://wicg.github.io/IntersectionObserver/#dom-intersectionobserverinit-rootmargin) allows you to specify the margins for the root, effectively allowing you to
+either grow or shrink the area used for intersections. These margins are
+specified using a CSS-style string, á la “`10px 20px 30px 40px`”, specifying top, right, bottom and left margin respectively. To summarize,
+the `IntersectionObservers` options struct offers the following options:
+
+{% highlight javascript %}
+new IntersectionObserver(entries => {/* … */}, {
+  // The root to use for intersection.
+  // If not provided, use the top-level document’s viewport.
+  root = null;
+  // Same as margin, can be 1, 2, 3 or 4 components, possibly negative lengths.  
+  // If an explicit root element is specified, components may be percentages of the
+  // root element size.  If no explicit root element is specified, using a percentage
+  // is an error.
+  rootMargin = "0px";
+  // Threshold(s) at which to trigger callback, specified as a ratio, or list of
+  // ratios, of (visible area / total area) of the observed element (hence all
+  // entries must be in the range [0, 1]).  Callback will be invoked when the visible
+  // ratio of the observed element crosses a threshold in the list.
+  threshold = [0];
+});
+{% endhighlight %}
+
+# iframe magic
+`IntersectionObservers` were designed specifically with ads services and social network widgets in
+mind, which frequently use iframes and could benefit from knowing whether they
+are in view. If an iframe observes one of its elements, both scrolling the
+iframe as well as scrolling the window *containing the iframe* will trigger the callback at the appropriate times. For the latter case,
+however, `rootBounds` will be set to `null` to avoid leaking data across origins.
+
+# What is `IntersectionObserver` *not* about?
```

Comment:
Remove the backticks:
![image](https://cloud.githubusercontent.com/assets/1749548/14686342/433a65be-0706-11e6-8845-0c0896f538bb.png)